Project.toml

```
@@ -43,7 +43,7 @@ OceananigansEnzymeCoreExt = "EnzymeCore"
 
 [compat]
 Adapt = "3"
-CUDA = "4, 5"
+CUDA = "^4.1.1, 5"
```

```suggestion
CUDA = "4.1.1, 5"
```

I'm pretty sure the `^` is redundant. https://pkgdocs.julialang.org/v1/compatibility/

---

Ah yes!